app/javascript/packs/pages/provider_signup/SignupStepper.jsx

```diff
@@ -135,17 +137,8 @@ export default function SignupStepper({ steps }) {
 
   const CurrentStep = steps[activeStep] && steps[activeStep].component;
 
-  if (activeStep === steps.length) {
-    return (
-      <Paper className={classes.paper}>
-        <Typography variant="h5" gutterBottom>
-          Thank you for signing up.
-        </Typography>
-        <Typography variant="subtitle1">
-          We have emailed you your account information.
-        </Typography>
-      </Paper>
-    );
+  if (redirectId) {
+    return <Redirect to={`/providers/${redirectId}?success`} />
```

Comment:
Thoughts on `?success` query param here to inform the ProviderPage to render the success message?

My only hesitation is that we will want to eventually update the success message to include a reference ID that I believe is intended to be private. Adding `?success` to any provider URL will thus surface this reference ID.

Reply:
This works for MVP. In a future state, the mutation could return the reference ID (`LinkedToken.token`), and the client could put it in local storage perhaps. Then the success route only renders if the local storage token matches the id param. This could all live in a route like `providers/:id/success`

Reply:
👍 

Makes sense.

Would also be curious to explore the `state` property from React Router's `<Redirect />` component:

https://reacttraining.com/react-router/web/api/Redirect